Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -109,9 +110,18 @@ class HomeFragmentViewModel @AssistedInject constructor(
                     }
                 }
             }
+            if (cwaSettings.lastAppVersion.value < BuildConfigWrap.VERSION_CODE &&
```

Comment:
Why do we need to check the onboarding values? If we are on the home screen, onboarding was completed in any case, wasn't it? Could we just check the last version for which the changelog was shown?

Reply:
I think we could get rid of the isOnboarded() check because as you said, this is always true anyways. But I'd leave the onboarding timestamp check in because that takes care of NOT displaying the release notes after the user completes onboarding.